doc/integrator/checker.rst

```diff
@@ -50,6 +50,29 @@ The checker use the following configuration structure:
         checker:
             forward_headers: ['Cookie', 'Authorisation']
 
+.. note::
+
+    The checker assumes that it can access the c2cgeoportal services via ``http://localhost``.
+    If this is not allowed on your server, you can override this behaviour as follows.
+    In the configuration file (see above), add the following:
```

Comment:
In your ``vars`` file ...

(same as below)

Reply:
OK, thanks